### DISCUSSION TITLE
Emulated Ethernet is VERY VERY SLOW

### DISCUSSION THREAD

**Discussion by adorni** [86Box/86Box#1590] [General]
Hi, i try 86Box. EXCELLENT. The only question is the NE2000. The NET (on DOS and /or Windows 95) works VERY SLOWLY. In PCEM works fine, but in 86box is very slow. Any idea???

Thanks

**Comment by dhrdlicka** [comment]
Do you have any exact values as to how slow it is, rather than "VERY VERY SLOW" and "slower than PCem"?

**Comment by OBattler** [comment]
Yes, and in PCem, it also slows down the entire emulator a lot. Why is why I capped the speed.

**Comment by adorni** [comment]
Is it possible, in the futures versions of 86Box, put the option for manage
the net speed???

Thanks

**Comment by Rasthro** [comment]
I have the same problem too, I have a Windows for workgroups 3.11 with all the important drivers (including network drivers) installed, The network was WAY TOO SLOW but it still worked fine

**Comment by adorni** [comment]
Yes, i have the same problem on any emulated machine.

**Comment by kode54** [comment]
Yeah, it works at dialup speeds.

**Comment by OBattler** [comment]
The network improvements haven't made it to release yet. So either wait patiently for the next release or get the latest nightly build from our Jenkins.
